src/customer_segmentation/SegmentAssigner.java

```diff
@@ -0,0 +1,27 @@
+package customer_segmentation;
+
+import java.math.BigDecimal;
+
+public class SegmentAssigner {
+
+    private static final BigDecimal VIP_INCOME = new BigDecimal("15000");
```

Comment:
please initialize the BigDecimal value using .valueOf(..) and make the numbers more readable [separating groups of digits](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html):
```suggestion
    private static final BigDecimal VIP_INCOME = BigDecimal.valueOf(15_000);
```

and please apply same logic to other BigDecimal initializations.